QuizApp/App/Screens/Quiz/QuizCell.swift

```diff
@@ -0,0 +1,106 @@
+import UIKit
+
+final class QuizCell: UICollectionViewCell {
+
+    var data: Category? {
+        didSet {
+            guard let data = data else { return }
+            update(data: data)
+        }
+    }
+
+    private var imageView: UIImageView!
+    private var titleLabel: UILabel!
+    private var descriptionLabel: UILabel!
+    private var difficultyView: DifficultyView!
+    static let reuseIdentifier = "QuizCell"
```

Comment:
use `static let reuseIdentifier = String(describing: QuizCell.self)` and move it to the top (read code style)